api/src/main/resources/workbench-api.yaml

```diff
@@ -377,8 +377,8 @@ paths:
         204:
           description: The user's initial credit expiration period has been extended.
           content: {}
-        500:
```

Comment:
we should probably keep 500 just becuz it's kinda like a catch all error (say, DB down or whatever can all cause 500)